### PR TITLE
feat: add logic / improve UX for `SaveBackupView`

### PR DESCRIPTION
**Pull request by maxwellmattryan**
## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

This PR is mostly UX polish for the flow.

## Changelog

```
- Please write a list of changes
```

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [x] Linux
    -   [ ] Windows
-   **Mobile**
    -   [ ] iOS
    -   [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [x] I have followed the contribution guidelines for this project
-   [x] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/shared/routes/setup/import/views/Success.svelte [hunk 1]
```diff
@@ -20,7 +20,7 @@
     }
 </script>
 
-<OnboardingLayout onBackClick={handleBackClick}>
+<OnboardingLayout allowBack={false}>
```
Comment (Tuditi) [inline packages/shared/routes/setup/import/views/Success.svelte:23]: You should remove the `handleBackClick()` function as well if we don't allow the user to go back.

**Review by Tuditi — CHANGES_REQUESTED** [review]
Requested some minor changes.

File: packages/shared/components/WarningBadge.svelte [hunk 2]
```diff
@@ -1,9 +1,12 @@
 <script lang="ts">
     import { Icon } from 'shared/components'
+
+    export let width = 38
+    export let height = 38
```
Comment (Tuditi) [inline packages/shared/components/WarningBadge.svelte:5]: Make these 32 and remove the props in `WarningBadge` :relaxed:

**Review by Tuditi — CHANGES_REQUESTED** [review]
I created some profiles with 1.7.1, but when I build this branch I receive the following error when I want to update the old profiles: 

`Stronghold migration error: Input snapshot has incorrect/unexpected version.`

**Review by Tuditi — CHANGES_REQUESTED** [review]
Additionally I also get an error from `checkChrysalisSnapshot()` on start-up:

```
migration.ts:1221 SyntaxError: Unexpected token : in JSON at position 3
```

**Comment by maxwellmattryan** [comment]
> Additionally I also get an error from checkChrysalisSnapshot() on start-up

Yeah that error has been there for a while I think - perhaps worth fixing in a separate PR. Will investigate though

**Comment by begonaalvarezd** [comment]
> Additionally I also get an error from `checkChrysalisSnapshot()` on start-up:
> 
> ```
> migration.ts:1221 SyntaxError: Unexpected token : in JSON at position 3
> ```

I get this same issue also in v1.7.1 so perhaps is unrelated to this PR

**Comment by begonaalvarezd** [comment]
> I created some profiles with 1.7.1, but when I build this branch I receive the following error when I want to update the old profiles:
> 
> `Stronghold migration error: Input snapshot has incorrect/unexpected version.`

I get the same issue if I only run yarn build && run start between versions, but you have to rebuild the entire backend, I usually run this from desktop to make sure I rebuild all `cd ../../ && yarn && cd packages/backend/bindings/node/ && yarn && cd ../../../desktop && yarn && yarn build && yarn start`, after running the full command, i dont get the above mentioned issue. It looks like you were generating profiles with the new stronghold version but firefly didnt know about it, so its trying to migrate an already migrated profile

File: packages/shared/routes/update-stronghold/UpdateStrongholdRouter.svelte [hunk 3]
```diff
@@ -6,6 +6,8 @@
 
     export let parentRouter: Router<unknown>
     export let isRecovery: boolean
+
+    let hasChangedPassword = false
```
Comment (begonaalvarezd) [inline packages/shared/routes/update-stronghold/UpdateStrongholdRouter.svelte:10]: I think `hasChangedPassword` this is not used anymore

**Review by begonaalvarezd — COMMENTED** [review]
Works great for me 🚀 
Just added a mini comment

**Comment by Tuditi** [comment]
> > I created some profiles with 1.7.1, but when I build this branch I receive the following error when I want to update the old profiles:
> > `Stronghold migration error: Input snapshot has incorrect/unexpected version.`
> 
> I get the same issue if I only run yarn build && run start between versions, but you have to rebuild the entire backend, I usually run this from desktop to make sure I rebuild all `cd ../../ && yarn && cd packages/backend/bindings/node/ && yarn && cd ../../../desktop && yarn && yarn build && yarn start`, after running the full command, i dont get the above mentioned issue. It looks like you were generating profiles with the new stronghold version but firefly didnt know about it, so its trying to migrate an already migrated profile

Yup, this seemed to be my issue. Works fine now for me :relaxed:

**Review by Tuditi — APPROVED** [review]
LGTM! :fire:

**Pull request merged**